M/Minuit2/build_tarballs.jl

```diff
@@ -31,6 +28,9 @@ if [[ "${target}" == *-mingw* ]]; then
     ar x libMinuit2.dll.a
     cc -shared -o "${libdir}/libMinuit2.dll" *.o
     rm *.o
+    ar x libMinuit2Math.dll.a
+    cc -shared -o "${libdir}/libMinuit2Math.dll" *.o
+    rm *.o
```

Comment:
Let's see if this finally works:
```suggestion
    cc -shared -o "${libdir}/libMinuit2.dll" $(bash flagon --whole-archive) libMinuit2.dll.a
    cc -shared -o "${libdir}/libMinuit2Math.dll" $(bash flagon --whole-archive) libMinuit2Math.dll.a
```

Reply:
Of course `flagon` is still broken: #2318.  Hopefully, next time we rebuild the RootFS it'll work